app/invites/client/admin/startup.js

```diff
@@ -6,6 +8,6 @@ AdminBox.addOption({
 	i18nLabel: 'Invites',
 	icon: 'user-plus',
 	permissionGranted() {
-		return hasAtLeastOnePermission(['create-invite-links']);
+		return hasAtLeastOnePermission(['create-invite-links'], Session.get('openedRoom'));
```

Comment:
the admin link doesn't need to check the permission for a specific scope
```suggestion
		return hasAtLeastOnePermission(['create-invite-links']);
```